src/content/meta/glossary.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Glossary
+description: A page with the term definitions Giant Swarm uses in the documentation.
+search: false
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - "What is a cloud-native developer platform?"
+  - "What is a developer environment?"
+  - "What is a management cluster?"
+  - "What is a managed app?"
+  - "What is the platform API?"
+  - "What is a platform operator?"
+  - "What is smart platform engineering?"
+  - "What is a workload cluster?"
+---
+
+We use a set of terms across our documentation that could be unclear to readers. In the following article we define those terms to clarify their meaning in the context of Giant Swarm.
+
+## Cloud native developer platform
+
+A cloud-native developer platform is an integrated collection of capabilities exposed through intuitive interfaces that facilitate the application development lifecycle. It offers flexible configuration to build a system that encodes your company practices. It integrates with different open source solutions to enable automation, connectivity, observability, and security for your workloads.
+
+## Developer environment
+
+In our context, a developer environment is the entity that defines the space where the developers will deploy their applications. It can be mapped to a Kubernetes namespace(s) or a cluster. The Platform capabilities like access management, security or observability are related to those environments.
+
+## Management cluster
+
+It is a Kubernetes cluster that acts a central point of the platform. It is in charge of hosting platform wide services like observability or security to ensure visualization and compliance over all the workloads in all clusters. [Platform operator](#platform-operator) have access to the its [API](#platform-api) to manage [workload clusters](#workload-cluster) and [managed apps](#managed-apps).
+
+## Managed app
```

Comment:
We decided quite some time ago to stop using the term "Managed app" and think more in terms of bundles/capabilities. @puja108 what do you think? How shall we call this here? "Managed services", "Managed capabilities", "Managed bundles"?

Reply:
Then what about the `App` resource which we'll have to describe? I think the term "app" in general is well-understood by users, so maybe we can combine the ideas somehow when we explain it (or make up a new heading/term).

Reply:
I am not against introducing a new concept but in the end, the app which will carry the capability or service will be there and we will manage it.

Reply:
Yes, we wanted to get rid of the managed app concept as it puts too much focus on apps and distracts from the actual capabilities they should enable. Among others, we wanted to avoid tech comparisons based on apps.

I prefer to use "managed capabilities" as services and bundles still have technical connotations.

That said, this does not mean that the App CR is not used here right now. It should, however, be kept in mind that that resource might go away, e.g. be replaced with something upstream like HelmRelease CR. The app in our case is an implementation detail of how we deploy helm charts declaratively. It can be defined separately, but then we're on an API level, not on a concept level like here.

I can do a suggestion that reflects the change here.